### ISSUE TITLE
Change `code()` helper so that it loads a file

### ISSUE THREAD

**Issue by davestewart** [davestewart/laravel-sketchpad#32]
Having to get file contents separately is cumbersome.

Would be better if it loaded a file, and you could grep a section, or specify start and end lines.

This might need to be class, not a helper.

**Comment by davestewart** [comment]
Possible API:

```
Code::method ($class, $name)
Code::file ($name, $lineStart = 0, $lineEnd = 0)
```

**Comment by davestewart** [comment]
Closed by 44d1c4b1bf6f305ca9eb4009514f689757de6e46
